src/concrete/ml/torch/compile.py

```diff
@@ -418,8 +414,11 @@ def compile_brevitas_qat_model(
     # https://github.com/onnx/optimizer/blob/master/onnxoptimizer/pass_registry.h
     # In the export function, the `args` parameter is used instead of the `input_shape` one in
     # order to be able to handle multi-inputs models
-    exporter.onnx_passes.append("eliminate_nop_pad")
-    exporter.onnx_passes.append("fuse_pad_into_conv")
+    exporter.onnx_passes += [
+        "eliminate_nop_pad",
+        "fuse_pad_into_conv",
+        "fuse_matmul_add_bias_into_gemm",
```

Comment:
This doesn't work, right?

Reply:
They work, but depending on the shape of the input.

Reply:
I see. But don't we have a function that does exactly that as well in CML? Should we keep both?

Reply:
`ceinture et bretelles expression` as we say

Reply:
As long as they don't interfere I am fine.